packages/fxa-content-server/app/scripts/views/settings/clients.js

```diff
@@ -286,31 +281,23 @@ const View = FormView.extend(
 
     _fetchAttachedClients() {
       const start = Date.now();
-      return this._attachedClients
-        .fetchClients(
-          {
-            oAuthApps: true,
-            sessions: true,
-          },
-          this.user
-        )
-        .then(() => {
-          // log the number of items
-          const numOfClients = this._attachedClients.length;
-          const itemsLog = 'settings.clients.items';
-
-          if (numOfClients === 0) {
-            this.logEventOnce(itemsLog + '.zero');
-          } else if (numOfClients === 1) {
-            this.logEventOnce(itemsLog + '.one');
-          } else if (numOfClients === 2) {
-            this.logEventOnce(itemsLog + '.two');
-          } else {
-            this.logEventOnce(itemsLog + '.many');
-          }
+      return this._attachedClients.fetchClients(this.user).then(() => {
+        // log the number of items
+        const numOfClients = this._attachedClients.length;
+        const itemsLog = 'settings.clients.items';
+
+        if (numOfClients === 0) {
```

Comment:
Are we making use of these events anywhere? I can't find any reference to them on the backend or fxa-shared, the only thing I see is a reference to [`settings.clients.disconnect.submit`](https://github.com/mozilla/fxa/blob/ae34698dee9370a411e88ab5e5361c965309ece6/packages/fxa-content-server/server/lib/amplitude.js#L175). This might be a remnant from DataDog days, and if so, might as well remove the events while you are digging around in here.

Reply:
I went ahead and removed them.